modules/hostap/src/supp_api.c

```diff
@@ -1091,7 +1091,7 @@ int supplicant_dpp_dispatch(const struct device *dev,
 
 	switch (params->action) {
 	case WIFI_DPP_CONFIGURATOR_ADD:
-		strncpy(pos, "dpp_configurator_add", end - pos);
+		strncpy(pos, "DPP_CONFIGURATOR_ADD", end - pos);
```

Comment:
fixup commits aren't allowed, they should be fixed up with the original commit.

Reply:
added in original commit